vortexasdk/api/shared_types.py

```diff
@@ -127,6 +127,13 @@ class Tag(BaseModel):
     start_timestamp: Optional[ISODate] = None
     end_timestamp: Optional[ISODate] = None
 
+    def to_json(self):
```

Comment:
I think it should enough to call dict on the model instance to get a dictionary https://docs.pydantic.dev/1.10/usage/exporting_models/#dictmodel-and-iteration

Reply:
There is also an inbuilt `json()` method if you prefer https://docs.pydantic.dev/1.10/usage/exporting_models/#modeljson